src/components/Login/LogoutButton.jsx

```diff
@@ -31,7 +37,9 @@ export class LogoutButton extends React.Component {
     }
     render() {        
         return (
+            (this.state.loggedIn) ?
```

Comment:
I think this stuff can be removed if we're wrapping up the whole Dashboard component now?

Reply:
That's when a person is already logged in and needs to get redirected to dashboard